dev-embedded/stlink/stlink-1.6.0.ebuild

```diff
@@ -17,13 +17,16 @@ fi
 
 LICENSE="BSD"
 SLOT="0"
-IUSE=""
 
 RDEPEND="virtual/libusb:1
 	>=dev-libs/glib-2.32.0:2
 	x11-libs/gtk+:3"
-DEPEND="${RDEPEND}
-	virtual/pkgconfig"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_prepare() {
+	cmake_src_prepare
+}
```

Comment:
drop this, and set `inherit udev xdg cmake`

Reply:
That looks nicer, thanks